battleship.py

```diff
@@ -214,6 +247,13 @@ def placeShip(data):
 Returns: None
 '''
 def clickUserBoard(data, row, col):
+    if data["num_User_ship"]==5:
+                print("you can start the game")
+                return
+    if [row, col] not in data["temporary_ship"]:
+                data["temporary_ship"].append([row, col])
+                if len(data["temporary_ship"])==3:
+                        placeShip(data)

```

Comment:
@mrudula-n The logic is quite ok. Check for the indentation of the statements in the bo the if statements.